rel/files/riak_cs.schema

```diff
@@ -200,6 +200,16 @@
   hidden
 ]}.
 
+%% @doc Small objects to be synchronously deleted including blocks on
+%% the fly. If this is used in combination with replication, cluster
```

Comment:
The work "replication" is used for inter-vnode replication as well as Multi-Datacenter. I think it means MDC in this case, adding MDC or some words is better.